### PR TITLE
Upgrade pVACtools CWLs to use version 1.4.4

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

**Comment by chrisamiller** [comment]
+1!

File: definitions/tools/pvacfuse.cwl [hunk 1]
```diff
@@ -21,7 +21,7 @@ arguments: [
 requirements:
     - class: ShellCommandRequirement
     - class: DockerRequirement
-      dockerPull: "griffithlab/pvactools:1.4.2"
+      dockerPull: "griffithlab/pvactools:1.4.3"
```
Comment (susannasiebert) [inline definitions/tools/pvacfuse.cwl]:
```suggestion
      dockerPull: "griffithlab/pvactools:1.4.4"
```

File: definitions/tools/pvacseq.cwl [hunk 2]
```diff
@@ -21,7 +21,7 @@ arguments: [
 requirements:
     - class: ShellCommandRequirement
     - class: DockerRequirement
-      dockerPull: "griffithlab/pvactools:1.4.2"
+      dockerPull: "griffithlab/pvactools:1.4.3"
```
Comment (susannasiebert) [inline definitions/tools/pvacseq.cwl]:
```suggestion
      dockerPull: "griffithlab/pvactools:1.4.4"
```

File: definitions/tools/pvacvector.cwl [hunk 3]
```diff
@@ -21,7 +21,7 @@ arguments: [
 requirements:
     - class: ShellCommandRequirement
     - class: DockerRequirement
-      dockerPull: "griffithlab/pvactools:1.4.2"
+      dockerPull: "griffithlab/pvactools:1.4.3"
```
Comment (susannasiebert) [inline definitions/tools/pvacvector.cwl]:
```suggestion
      dockerPull: "griffithlab/pvactools:1.4.4"
```

**Comment by susannasiebert** [comment]
1.4.4 is building: https://cloud.docker.com/u/griffithlab/repository/registry-1.docker.io/griffithlab/pvactools/builds/8a8aea65-5a32-4bd2-9926-e0819d6d508e

**Pull request merged**